examples/pytorch/rgcn/entity_classify_mp.py

```diff
@@ -541,7 +543,7 @@ def main(args, devices):
     n_gpus = len(devices)
     n_cpus = mp.cpu_count()
     # cpu
-    if devices[0] == -1:
```

Comment:
Why we need n_gpus == 1? This is for CPU training. The device is set to ['cpu']

Reply:
I was doing some multi-socket CPU only testing (`--gpu="-1,-1"`), and this would break. I can remove it if you want.

Reply:
I c.
My suggestion is keepping it unchanged.

Reply:
Reverted in 
01c653f